Sensor/toon.py

```diff
@@ -30,7 +40,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
                          '{}_today_energy'.format(plug.name),
                          plug.name,
                          'kWh')])
-    if _toon_main.toon.solar.value:
+
+    if _toon_main.toon.solar.value or _toon_main.solar:
```

Comment:
I think its better here if we check for "produced" and not value. Since I don't have solar I cannot validate this but I am thinking that it might be that solar.value could be 0 if there is no sun which does not mean that it is not installed whereas checking for the produced would return 0 if there is no solar installed.